# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 3a34d557d474
# Full Hash: 3a34d557d474adff2245632b993529dd8e01cb3a
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move DOM module loader into own file; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137531
# ==============================================================================

diff -r f29451ea18dc -r 3a34d557d474 dom/script/moz.build
--- a/dom/script/moz.build	Thu Feb 17 15:57:50 2022 +0000
+++ b/dom/script/moz.build	Thu Feb 17 15:57:51 2022 +0000
@@ -20,6 +20,7 @@
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
     "LoadedScript.h",
+    "ModuleLoader.h",
     "ModuleLoaderBase.h",
     "ModuleMapKey.h",
     "ScriptDecoding.h",
@@ -33,6 +34,7 @@
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
     "LoadedScript.cpp",
+    "ModuleLoader.cpp",
     "ModuleLoaderBase.cpp",
     "ModuleLoadRequest.cpp",
     "ModuleMapKey.cpp",
